save-sandbox/src/main/resources/application-dev.properties

```diff
@@ -1,4 +1,8 @@
 sandbox.backendUrl=http://localhost:5800/internal
 sandbox.docker.logging-driver=json
 sandbox.docker.test-resources-volume-name=save-tmp-resources
-sandbox.docker.test-resources-volume-type=bind
\ No newline at end of file
+sandbox.docker.test-resources-volume-type=bind
+spring.datasource.url=jdbc:mysql://localhost:3306/save_sandbox
+datasource.dev.url=jdbc:mysql://localhost:3306/save_sandbox
```

Comment:
I think this property is unused, it was only used from backend's application.properties

Reply:
also it's equal to `spring.datasource.url` there. @petertrr can we remove this property?

Reply:
I think so. Originally we planned to use this property to switch easier from running as a standalone JVM process and running inside the container, but I believe we have more relevant processes now. I think the only place where it was in use is getting credentials for local database.